db/db_batch.go

```diff
@@ -28,22 +28,20 @@ type (
 	KVStoreBatch interface {
 		// Lock locks the batch
 		Lock()
-		// Unlock unlocks the batch
-		Unlock()
+		// ClearAndUnlock clears the write queue and unlocks the batch
+		ClearAndUnlock()
 		// Put insert or update a record identified by (namespace, key)
-		Put(string, []byte, []byte, string, ...interface{}) error
```

Comment:
Put() just adds a new entry into the buffer, error is not required
so does Delete()